spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/BinderAwareChannelResolverTests.java

```diff
@@ -74,7 +74,7 @@
 
 	private volatile BinderAwareChannelResolver resolver;
 
-	private volatile Binder<MessageChannel, ConsumerProperties, ProducerProperties> binder;
+	private volatile ExtendedPropertiesBinder<MessageChannel, ExtendedConsumerProperties, ExtendedProducerProperties> binder;
```

Comment:
While we need to test the new change, we should create a separate test for it - now we don't have a test for a normal (non-extended) binder.